styles.css

```diff
@@ -65,6 +83,248 @@ nav li a img {
   background-color: white;
 }
 
+@media screen and (min-width: 768px) {
```

Comment:
I have adjusted the Design and used %Percentages on images instead of defining width in Pixels. Now The Design in not breaking at that exact particular viewpoint which you mentioned above. Attaching the screen shots for your kind reference and will then send for a re-review.

![image](https://user-images.githubusercontent.com/117091923/215055618-151b0c48-ff1f-4a89-ae3d-ab409e314347.png)